launchable/test_runners/nunit.py

```diff
@@ -76,38 +81,37 @@ def split_filepath(path: str) -> List[str]:
 
         # "Assembly" type contains full path at a customer's environment
         # remove file path prefix in Assembly
-        e.tags['path'] = [
+        e.set('path', unparse_test_path([
             {**path, 'name': split_filepath(path['name'])[-1]}
             if path['type'] == 'Assembly'
             else path
             for path in pp
-        ]
+        ]))
 
 
 def nunit_parse_func(report: str):
     events = []
 
     # parse <test-case> element into CaseEvent
-    def on_element(e: Element):
-        build_path(e)
-        if e.name == "test-case":
-            result = e.attrs.get('result')
+    def on_element(e: ET.Element, parent: ET.Element):
+        build_path(e, parent)
+        if e.tag == "test-case":
+            result = e.attrib.get('result')
             status = CaseEvent.TEST_FAILED
             if result == 'Passed':
                 status = CaseEvent.TEST_PASSED
             elif result == 'Skipped':
                 status = CaseEvent.TEST_SKIPPED
 
             events.append(CaseEvent.create(
-                _replace_fixture_to_suite(e.tags['path']),  # type: ignore
-                float(e.attrs['duration']),
+                _replace_fixture_to_suite(e.attrib['path']),  # type: ignore
+                float(e.attrib['duration']),
                 status,
-                timestamp=str(e.tags['startTime'])))  # timestamp is already iso-8601 formatted
+                timestamp=str(e.attrib.get('start-time'))))  # timestamp is already iso-8601 formatted
 
     # the 'start-time' attribute is normally on <test-case> but apparently not always,
```

Comment:
Do we need this comment here?

Reply:
[ec20f06](https://github.com/launchableinc/cli/pull/641/commits/ec20f06440ca75dfb58bfc441e15ea4a130eaa00)